tox.ini

```diff
@@ -85,7 +85,7 @@ deps =
     -r{toxinidir}/requirements.txt
     deepdiff
 commands =
-    sh -c 'stat promql-transform > /dev/null 2>&1 || curl "https://github.com/canonical/promql-transform/releases/download/2.25.2-1rc1/promql-transform_2.25.2-1rc1_linux_amd64.tar.gz" -L -s | tar zxv promql-transform'
+    sh -c 'stat promql-transform-amd64 > /dev/null 2>&1 || curl -L -O https://github.com/canonical/promql-transform/releases/download/rev6/promql-transform-amd64'
```

Comment:
pytest operator would still build the charm with the charmcraft yaml parts so do we still need these curls in tox?

Reply:
Nope, not for integration. We do for unit, though.